dotCMS/src/main/java/com/dotmarketing/db/listeners/CommitListener.java

```diff
@@ -0,0 +1,28 @@
+package com.dotmarketing.db.listeners;
+
+public abstract class CommitListener implements DotListener {
+
+    @Override
+    public final boolean equals(Object obj) {
+        
+        
+        if(obj == null ||!( obj instanceof CommitListener)) {
+            return false;
+        }
+        CommitListener other = (CommitListener)obj;
+        if(this.key() ==null ) {
+            return false;
+        }
+        return this.key().toLowerCase().equalsIgnoreCase(other.key());
```

Comment:
Codacy found an issue: [Using equalsIgnoreCase() is cleaner than using toUpperCase/toLowerCase().equals().](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)